third_party/README.md

```diff
@@ -118,7 +118,10 @@ Apart from the above steps, please perform the following two steps to generate t
 Since upb is vendored in the gRPC repo, you cannot use submodule to update it. Please follow the steps below.
 
 1. Update third_party/upb directory by running
-   `git subtree pull --squash --prefix=third_party/upb https://github.com/protocolbuffers/upb.git main`
+   - `wget https://github.com/protocolbuffers/upb/archive/refs/heads/main.zip`
+   - `rm -rf  ~/git/grpc/third_party/upb`
```

Comment:
note that the `~/git/grpc` part is specific to your personal setup, might be different for others. E.g. set variable GRPC_ROOT= at the beginning so others can easily customize?